apps/api/src/billing/services/managed-signer/managed-signer.service.ts

```diff
@@ -24,6 +24,8 @@ import { TrialValidationService } from "../trial-validation/trial-validation.ser
 
 type StringifiedEncodeObject = Omit<EncodeObject, "value"> & { value: string };
 
+const SPENDING_TXS = [MsgCreateDeployment, MsgAccountDeposit, MsgCreateLease];
```

Comment:
Technically MsgCreateLease doesn't affect the balance of the escrow

Reply:
ok, removing